valusets/ValueSet-England-DMMedicationRequestCategory.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+	<id value="England-DMMedicationRequestCategory"/>
+	<url value="https://fhir.nhs.uk/ValueSet/England-DMMedicationRequestCategory"/>
+	<version value="1.0.0"/>
+	<name value="EnglandDMMedicationRequestCategory"/>
+	<title value="England DM MedicationRequest Category" />
```

Comment:
Is the title "England DM Medication Request Category" or "England DM MedicationRequest Category"?

Reply:
According to the current understanding of our formatting approach, I think England DM MedicationRequest Category is correct. Please correct me if I am wrong in this.